src/main/kotlin/domain/Count.kt

```diff
@@ -0,0 +1,17 @@
+package domain
+
+@JvmInline
+value class Count(val value: Int) {
+    init {
+        require(value >= MINIMUM_NON_NEGATIVE_NUMBER) { COUNT_NEGATIVE_ERROR_MESSAGE }
+    }
+
+    operator fun plus(number: Int): Count {
+        return Count(this.value + number)
+    }
+
+    companion object {
+        const val COUNT_NEGATIVE_ERROR_MESSAGE = "[ERROR] 갯수는 음수가 될 수 없습니다."
+        const val MINIMUM_NON_NEGATIVE_NUMBER = 0
```

Comment:
```suggestion
        private const val MINIMUM_NON_NEGATIVE_NUMBER = 0
```
클래스 내부에서만 사용하는 변수라면, private 접근자를 붙여보면 어떨까요?
private 접근자가 없으면 외부에서 접근이 가능하게 되고, 그렇게 되면 해당 변수값의 변경으로 인하여 의도치 않게 다른 클래스에 영향을 미치게 됩니다.
가급적이면 private를 붙여서 작성하고, 외부에서 접근하는 경우에 private를 지우는 습관을 들이는 것을 권장합니다. (이하 관련 내용 동일)

Reply:
수정 하였습니다!